src/control/controlpotmeter.cpp

```diff
@@ -44,6 +55,14 @@ void ControlPotmeter::setRange(double dMinValue, double dMaxValue,
     }
 }
 
+void ControlPotmeter::checkValue(double v) {
+    if (v == defaultValue()) {
```

Comment:
first I tried to avoid the `ControlObject > defaultValue()` lookup on every value change by storing defaultValue when ControlPotmeter is initialized, but that wouldn't work for ControlPotmeter derivates that set their own default, like `ControlAudioTaperPot` "volume" for example.

> `ControlObject > defaultValue()` lookup on every value change

is that actually a performance issue?

Reply:
Not really. I will have a look If there is an alternative.

Reply:
mmm, this is a performance issue if the value is changed from another thread, in this case the checkValue() is called via a queued connection. 
We can clalc the default state here and emit a signal only for the case it changes: https://github.com/mixxxdj/mixxx/blob/fa1369f880494d1e3c9aa7f41c66f2aae8212741/src/control/control.cpp#L222
This will affect "EVERY" CO though ...

Reply:
I think the best is to use a direct connection to ControlDoublePrivate::valueChanged() this way it is done in the thread that set the value. Like here: 
https://github.com/mixxxdj/mixxx/blob/fa1369f880494d1e3c9aa7f41c66f2aae8212741/src/control/controlobject.cpp#L50

If my thought where to confusing, I am happy to propose a PR for this change.

Reply:
Not confusing but I'm a bit stingy about my spare time available for Mixxx.
Please send a PR (or push(-force) directly if you already have a working solution ready to hand.